src/__tests__/starWarsData.js

```diff
@@ -1,10 +1,12 @@
+/* @flow */
+/* eslint quote-props: ["error", "as-needed",
+{ "keywords": true, "unnecessary": false }]*/
```

Comment:
I think you actually want `{numbers: true}` rather than `keywords` to handle this case.

I just landed a change which applies this across the whole project so that we would not need this special case.